### PR TITLE
Gdv 206 GLOFFIS data hydroengine

### PR DESCRIPTION
**Pull request by ckrogers**

### CODE REVIEW

File: ansible/roles/frontend/tasks/main.yml [hunk 1]
```diff
@@ -7,29 +7,15 @@
...
 - name: delete frontend zip
-  file: path=/tmp/dgds/ state=absent
-  become: yes
-
-- name: delete frontend from tmp
-  file: path=/tmp/dist state=absent
-  become: yes
-  
+  file: path=/tmp/dgds-ui.zip state=absent
+  become: yes
```
Comment (avgils) [inline ansible/roles/frontend/tasks/main.yml]: wrong merge? this should not change

File: ansible/roles/frontend/tasks/main.yml [hunk 2]
```diff
@@ -7,29 +7,15 @@
 
 - name: unzip frontend
   unarchive:  src=/tmp/dgds-ui.zip
-              dest=/tmp/
+              dest={{ app_folder }}/static
```
Comment (avgils) [inline ansible/roles/frontend/tasks/main.yml]: wrong merge? this should not change

File: ansible/roles/nginx/templates/nginx.conf.j2 [hunk 3]
```diff
@@ -28,15 +28,15 @@ server {
     # max upload size
     client_max_body_size 1M;
 
-    root  {{ app_folder }}/static;
+    root  {{ app_folder }}/static/dist;
```
Comment (avgils) [inline ansible/roles/nginx/templates/nginx.conf.j2]: wrong merge?

File: ansible/roles/nginx/templates/nginx.conf.j2 [hunk 4]
```diff
@@ -28,15 +28,15 @@ server {
...
     location /api {
       include uwsgi_params;
       uwsgi_pass dgds;
     }
 
     location / {
-        try_files $uri /index.html;
-        index index.html;
+        try_files $uri /200.html;
+        index  200.html;
```
Comment (avgils) [inline ansible/roles/nginx/templates/nginx.conf.j2]: wrong merge?

File: dgds_backend/config_data/datasets.json [hunk 5]
```diff
@@ -151,6 +151,41 @@
...
+		{
+			"id": "dd",
+			"name": "Discharge",
+			"units": "m3/s",
+			"timeSpan": "Live",
+			"themes": ["fl"]
+		},
+		{
+			"id": "ro",
+			"name": "Run-off Simulated",
```
Comment (avgils) [inline dgds_backend/config_data/datasets.json]: This was not in the email of Daniel. Not sure if we should add this one.

File: dgds_backend/config_data/datasets.json [hunk 6]
```diff
@@ -151,6 +151,41 @@
...
+			"units": "mm",
+			"timeSpan": "Live",
+			"themes": ["fl"]
+		},
+		{
+			"id": "so",
+			"name": "Soil Moisture",
```
Comment (avgils) [inline dgds_backend/config_data/datasets.json]: same as run-off simulated. This parameter is also not in de email of Daniel.

File: dgds_backend/config_data/datasets_access.json [hunk 7]
```diff
@@ -59,9 +57,7 @@
       "name": "waterlevel",
       "protocol": "hydroengine",
       "parameters": {
-        "bandNames": [
-           "water_level_surge"
-         ]
+        "bandNames": "water_level_surge"
```
Comment (avgils) [inline dgds_backend/config_data/datasets_access.json]: bandNames -> plural, but it is a single paramter. Why string not a list?

File: dgds_backend/config_data/datasets_access.json [hunk 8]
```diff
@@ -126,5 +122,85 @@
...
+      "name": "",
+      "protocol": "",
+      "parameters": {}
+    },
+    "rasterService": {
+      "url": "https://dgds-test-dot-hydro-engine.appspot.com/get_gloffis_data",
+      "name": "hydro",
+      "protocol": "hydroengine",
+      "parameters": {
+        "bandNames": "runoff_simulated"
```
Comment (avgils) [inline dgds_backend/config_data/datasets_access.json]: same here. This paramter was not requested by Daniel

File: dgds_backend/config_data/datasets_access.json [hunk 9]
```diff
@@ -126,5 +122,85 @@
...
+      "name": "",
+      "protocol": "",
+      "parameters": {}
+    },
+    "rasterService": {
+      "url": "https://dgds-test-dot-hydro-engine.appspot.com/get_gloffis_data",
+      "name": "hydro",
+      "protocol": "hydroengine",
+      "parameters": {
+        "bandNames": "soil_moisture"
```
Comment (avgils) [inline dgds_backend/config_data/datasets_access.json]: This parameter is not requested by Daniel

File: tests/test_dgds_backend.py [hunk 10]
```diff
@@ -66,7 +66,7 @@ def test_get_hydroengine_url(self, mock_post):
         id = "cc"
         layer_name = "currents"
         access_url = "https://sample-hydro-engine.appspot.com/get_glossis_data"
-        parameters = {"bandNames": []}
+        parameters = {"bandNames": ""}
```
Comment (evetion) [inline tests/test_dgds_backend.py]: What changed in the specification?

**Comment by evetion** [comment]
See #24

**Pull request closed without merging**